src/app/shared/page/footer/footer.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styles: [
+    `
+      @media (min-width: 768px) {
+        .text-center {
+          text-align: right !important;
+        }
+      }
+    `,
+  ],
```

Comment:
use class `text-center md:text-start` instead

Reply:
I tried it before and it did not work. I tracked the class and I think it's conflict from Bootstrap